luda-editor/luda-editor-client/src/app/app.rs

```diff
@@ -1,43 +1,60 @@
-use super::{router::RouterProps, types::AppContext, Router};
+use super::{
+    router::RouterProps,
+    types::{
+        meta::{self, *},
+        AppContext,
+    },
+    Router,
+};
+use async_trait::async_trait;
+use luda_editor_rpc::Socket;
 use namui::Wh;
+use std::sync::Arc;
 use wasm_bindgen_futures::spawn_local;
 
 pub struct App {
     router: Router,
+    meta_container: Arc<MetaContainer>,
 }
 
 impl namui::Entity for App {
     type Props = ();
     fn render(&self, _: &Self::Props) -> namui::RenderingTree {
-        let screen_size = namui::screen::size();
-        self.router.render(&RouterProps {
-            screen_wh: Wh {
-                width: screen_size.width as f32,
-                height: screen_size.height as f32,
-            },
-        })
+        match self.meta_container.get_meta() {
```

Comment:
If meta_container didn't load meta yet, it won't render. I didn't process about the error that occurs during meta loading.